doc/gov/chapters/chapter05.md

```diff
@@ -33,8 +33,19 @@ Being on the voting representative list **doesn't** mean the right to vote. The
 
 <sup>2)</sup>The criteria will be checked and the voting representative list will be updated prior any voting event when the voting process kicks in.
 
-<sup>3)</sup> Individuals who don't present an organisation will not have voting right.
+<sup>3)</sup> Individuals who don't represent an organisation will not have voting right.
 
 <a name="5.3"></a>
 ## 5.3 Voting Process
-
+ 
+ In Scenarios where Consensus cannot be reached, a voting event will be triggered by Technical/Governance Steering Leads (depending on if the event corresponds to a technical or governance matter). 
+
+ When a voting event is triggered, an online voting mechanism will be setup and the following process will be followed:
+ 
+ - The time and the deadline of which the voting representatives are required to cast their vote will be determined by the leads themselves.
```

Comment:
The time and deadline before which the voting ....

Reply:
fixed in https://github.com/cntt-n/CNTT/pull/897/commits/bcb48e960df22a1cf990e190c8567717e11806be